assignments/callbacks.js

```diff
@@ -24,29 +24,54 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
+getLength(items, function(item) {
+  console.log(item);
+});
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length - 1]);
 }
+last(items, function(item) {
```

Comment:
great job! if you're going to reuse the same structure callback like this throughout a file, i suggest just going ahead and defining it as a function at the top - function callback(input) {console.log(input)} - and just passing it to the functions like "last(items, callback)" so you don't have to keep defining it in your calls.